src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -190,7 +191,7 @@ const ScheduleActivityPage = ({isLoggedIn, accessToken, userId, activity, links,
       }
 
       <div className={classes.root}>
-      <Button variant="contained" color="primary" className={classes.largeButton} 
+      <Button variant="contained" color="primary" style={custom.largeButton} 
         onClick={handleSubmit}>Create Event</Button>
```

Comment:
Either shift the button declaration to the same line or three different lines to enhance readability. 

Button tag
  Create event
Closing button tag

or

Button tag  Create Event Closing Button tag	      

Note: I am not able to write code i don't know why haha, but I hope you got the idea.

Reply:
Yeah, I agree. Fixed it.